qtgui/settingutil.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SETTINGUTIL_H
+#define SETTINGUTIL_H
+#include <QSettings>
+
+//TODO: сделать ui для настройки
+namespace andeme {
+class settingutil{
```

Comment:
```suggestion
class Settings {
```
Лучше просто Settings назвать. На Util не тянет. util - это обычно просто набор статических методов, без хранения состояния.